tests/framework/db/mysql/SchemaTest.php

```diff
@@ -34,7 +49,13 @@ public function constraintsProvider()
         $result['3: foreign key'][2][0]->foreignTableName = new AnyCaseValue('T_constraints_2');
         $result['3: check'][2] = false;
 
+        // https://github.com/yiisoft/yii2/issues/15248
+        if (version_compare($this->getServerVersion(), '8.0.0', '>=') === true) {
```

Comment:
I do not really understand the underlying issue, but it looks wrong to me to add version check logic to tests, just to make them pass. Although I can not provide another solution yet.

Reply:
I think we have error in MySQL. We can't change this(